src/test/java/core/basesyntax/service/ReaderServiceTest.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.service;
+
+import static java.nio.file.Files.readAllLines;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReaderServiceTest {
```

Comment:
Make a constant path into reader services - very bad idea and bad practice. Cause now all your tests here are useless.